server.js

```diff
@@ -1,6 +1,7 @@
 const Discord = require("discord.js");
 const _ = require("lodash");
 const utils = require("./utils");
+require('dotenv').config()
```

Comment:
This is not listed in the package.json - is it really needed for the bot or is it something that you use locally for development?